Jenkinsfile

```diff
@@ -285,7 +258,7 @@ def checkoutPrBranchInCiviHRRepos(String branch) {
   for (repo in listCivihrGitRepoPath()) {
     try {
         sh """
-          cd ${repo}
+          cd ${repo.folder}
```

Comment:
If `listCivihrGitRepoPath` returns CiviHR and T&A repos, but the PR branch only existing in CiviHR will `git checkout ${branch}` cause an error? I guess that's what the try catch block is for, but it's done differently above with `git checkout $envBranch || true`

Reply:
You're right. The `try/catch` and the `|| true` are doing exactly the same thing. I'll update the file to use `|| true` here as well, as it's a bit more concise